enos/enos-scenario-package-manager.hcl

```diff
@@ -0,0 +1,217 @@
+scenario "package_manager" {
+  matrix {
+    arch            = ["amd64", "arm64"]
+    backend         = ["consul", "raft"]
+    artifact_source = ["local", "crt", "artifactory"]
+    consul_version  = ["1.13.2"]
+    distro          = ["ubuntu", "rhel"]
+    edition         = ["ent"]
```

Comment:
Reporting back from asking the team about this -- we do not currently because it was not part of the initial round of requests for enos scenario coverage, but it's something we'd like to add for sure.